contracts/src/tokens/memecoin.cairo

```diff
@@ -120,6 +134,9 @@ mod UnruggableMemecoin {
         }
 
         fn transfer(ref self: ContractState, recipient: ContractAddress, amount: u256) -> bool {
+            if (recipient == self.ownable.owner()) {
```

Comment:
Is this really necessary? That means the team bought back their own coin 🤔  If they wanted to go around this they would just transfer to another address